docs/book/src/reference/watching-resources/externally-managed.md

```diff
@@ -0,0 +1,31 @@
+# Watching Externally Managed Resources
+
```

Comment:
imo the goal to use watch is re-triger the reconcile when the k8s api raise an event such as create/update/edit/delete of an action performed against the resource observed. 

For the kubebuilder does I do not think that we should describe detailed scenario such as this one which can lead a misunderstands. We can be more direct ad object such as:

- You want to watch a resource that is managed by the controller then ...

- Add  a note about what means managed by the  controller as you explained with the owner  (see in the other docs the syntax to add notes)
- Add the example and a very low verbose explation

- You want to watch a resource that is NOT managed by the controller then ...
- You want to implement a customized watch for specific scenarios then ...

So, what I am trying to say is that we ought to not cover when each scenario is or not appropriate because it is up to the solution and requirements.

Reply:
Ok, made the changes you suggested for wording in `watching-resources.md`, as well as fleshed out some information on externally managed vs operator managed.

> So, what I am trying to say is that we ought to not cover when each scenario is or not appropriate because it is up to the solution and requirements.

I read through the examples, and I'm not exactly sure where it says when its appropriate to use each. I give some examples of the Kubernetes resources using the paradigms, then I briefly explain what the example code is trying to accomplish.

I think it would be much more confusing for users to not understand what the code is trying to do before trying to read through it.

Would you feel better about it if I emphasized more that this is merely an example, and not necessarily the right design choice for this situation?